### ISSUE TITLE
`calibration` as observationType

### ISSUE THREAD

**Issue by peterdesmet** [tdwg/camtrap-dp#216]
_Originally reported by @PatrickAJansen at https://github.com/inbo/camtraptor/issues/96#issue-1178267448 and split into 3 issues._

It would be logical and practical for users to adopt three additional values of observationType in observations.csv (3 of 3):

(3) "calibration", for sequences with markers placements for the purpose of calibration.
Currently, these sequences are recorded either as observationType="human", or as observationType="unclassified" with cameraSetup="TRUE". It is important that these sequences can be distinguished as calibration sequences.

**Comment by peterdesmet** [comment]
See also #215 regarding including `setup/pickup` as observationType. `calibration` can be argued as a specific subcategory of camera setup and I understand the desire to be able to search/filter on these, which is currently not possible with the boolean field `cameraSetup`.

Pending the discussion in #215 it could be added as a (sub) category of `setup` in `observationType`. An alternative would be to make `cameraSetup` a field with categories on its own (rather than a boolean field), but I'd prefer the `observationType` solution.

**Comment by ben-norton** [comment]
Why are these designations necessary?  In other words, what is the use case for the following observationTypes:
setup
pickup
calibration
These are actions instead of entities in an image. By including any of the terms listed above, we would be mixing concepts, which could lead to an array of problems. 

I think this problem could be resolved much more quickly if the parent term 'observation' is explicitly defined. Then we'll know whether something is a "type of". 

In terms of use cases, this is a more narrow question. Why is it necessary, within the context of an observation, to differentiate between setup, pickup, calibrations, and so on?
I can think of one use case off-hand.
1. This is often used as markers for the start and end date-times of a deployment. Verification of a start and end date is probably a better use, but either way, there's no reason for differentiating the setup, pickup, and calibration actions. Start and end will suffice. 

In summary, two questions.
1. Is it necessary to distinguish between setup, pickup, calibration etc. within the context of sharing camera trap observations?
2. Should a set of actions be grouped with identified entities under a single field called observationType?

**Comment by kbubnicki** [comment]
Same conclusions as in #215. I this one can be closed.

**Comment by peterdesmet** [comment]
Indeed, see action items at https://github.com/tdwg/camtrap-dp/issues/215#issuecomment-1425643398
